lib/trino-filesystem-gcs/src/test/java/io/trino/filesystem/gcs/TestGcsFileSystemConfig.java

```diff
@@ -70,7 +70,7 @@ void testExplicitPropertyMappings()
                 .put("gcs.json-key", "{}")
                 .put("gcs.json-key-file-path", jsonKeyFile.toString())
                 .put("gcs.client.max-retries", "10")
-                .put("gcs.client.backoff-scale-factor", "3.0")
+                .put("gcs.client.backoff-scale-factor", "4.0")
```

Comment:
What does it fix?

Reply:
The default was changed to `3.0` recently, but the explicit mapping was not. The test would fail.